WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialBannerPresenter.swift

```diff
@@ -134,12 +134,9 @@ private extension FreeTrialBannerPresenter {
 
     /// Shows a web view for the merchant to update their site plan.
     ///
-    func showUpgradePlanWebView() {
+    func showSubscriptionsView() {
         guard let viewController else { return }
-        let upgradeController = UpgradePlanCoordinatingController(siteID: siteID, source: .banner, onSuccess: { [weak self] in
-            self?.removeBanner() // Removes the banner immediately.
-            self?.reloadBannerVisibility() // Reloads the plan again in case the plan didn't update as expected.
-        })
-        viewController.present(upgradeController, animated: true)
+        let upgradeController = UpgradesHostingController(siteID: siteID)
+        viewController.show(upgradeController, sender: true)
```

Comment:
This change here hints that the `UpgradesHostingController`, `UpgradesView`, and `UpgradesViewModel` should all be updated to contain the `Subscription` wording instead. But given the criticality of this PR changing many things simultaneously, maybe we should update this later in a separate PR. Let me know if you think otherwise.

Reply:
Maybe add a comment that it will be renamed later?

Reply:
Indeed! Added here: https://github.com/woocommerce/woocommerce-ios/pull/9627/commits/f55da40d133204c7a8d5c300111a568f8c0e1b2f